plugins/node/opentelemetry-instrumentation-restify/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@opentelemetry/instrumentation-restify",
+  "version": "0.14.0",
```

Comment:
```suggestion
  "version": "0.15.0",
```

Reply:
This is outside the scope of this PR, but could we potentially remove the need for keeping this in sync with `./src/version.ts`?
Updated it.

Reply:
the `version.ts` is being generated based on the version from package.json during compilation, did you create the `version.ts` manually ?

Reply:
> This is outside the scope of this PR, but could we potentially remove the need for keeping this in sync with ./src/version.ts?

As bart said this is automatically generated when publishing new versions, we generally ask them to be up to date for new instrumentations so people doesn't get confused when looking at unpublishe code

Reply:
I did create it manually, by copying another instrumentation before I started. Should I remove it?

Reply:
Its not really important as long as version are sync between the version and the entry in the package.json, so i dont think so